tests/activemq5-unit-tests/src/main/resources/stomp/META-INF/services/org.apache.activemq.artemis.spi.core.protocol.ProtocolManagerFactory

```diff
@@ -0,0 +1 @@
+org.apache.activemq.artemis.core.protocol.stomp.StompProtocolManagerFactory
```

Comment:
Adding this file under testsuite would be a hack.

The problem with that is if we ever make any changes, or need any fix.. nobody will figure out what happened here.


Please remove this file from the testsuite, and if you need stomp protocol manager just add the proper dependency through the pom.